src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,37 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ELEMENTS_COUNT = 10;
+    private K[] keys = (K[]) new Object[ELEMENTS_COUNT];
+    private V[] values = (V[]) new Object[ELEMENTS_COUNT];
+    private int size = 0;
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if (key == null ? keys[i] == null : key.equals(keys[i])) {
+                values[i] = value;
+                return;
+            }
+        }
+        keys[size] = key;
+        values[size] = value;
+        size++;
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < size; i++) {
+            K k = keys[i];
+            if (key == null ? keys[i] == null : key.equals(keys[i])) {
```

Comment:
This object is redundant
```suggestion
        for (int i = 0; i < size; i++) {
            if (key == null ? keys[i] == null : key.equals(keys[i])) {
```